core/forms.py

```diff
@@ -91,7 +89,7 @@ def confirm_login_allowed(self, user):
         if not user.is_active:
             case_id = str(uuid4()).upper()
             auth_log.warning(
-                "User '{u.username}' tried to log in, but the account is deactivated [{cid}]."
+                (self.admin_inactive_user_notification + ", but the account is deactivated [{cid}].")
                 .format(u=user, cid=case_id)
             )
```

Comment:
```py
auth_log.warning(f"{self.admin_inactive_user_notification}, but the account is deactivated [{case_id}].")
```

Reply:
He, ĉu tio funkcios? Ĉar `admin_inactive_user_notification` mem enhavas anstataŭigendan ĉenon. Ĉu `f"..."` farus tian du-nivelan anstataŭon?

Reply:
Eble `f"{f'{self.admin_inactive_user_notification}'}, but the account is deactivated [{case_id}]."` ;-)

Reply:
Pardonu, mi pensis ke estis iu `@property` ie.  
Kial estas klasa variablo? Ĉu estas uzata alie? Aspektas strange tiel.

Reply:
vidu `SystemPasswordResetRequestForm` kaj `UsernameRemindRequestForm` en la sama dosiero. `admin_inactive_user_notification` estas uzitaj por aldoni al la protokolo la ĝustan agon de la uzanto. ĉu estas pli taŭga maniero atingi la saman rezulton?

Reply:
Malfacilas vidi tion en la Github interfaco, bone. Vi povas lasi tiel. Mia impreso estas ke estas iom tro da logiko en `send_mail`. Eble apartigi tion en _metodon_. Sed nun ne gravas.